.github/workflows/zip-release.yml

```
@@ -13,9 +13,11 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
       - name: Zip logos for domjudge
-        run: cd ${{ github.workspace }}; mkdir out;cd organizations; for f in *; do cp "$f/logo.64.png" "../out/$f.64.png"; done; cd ../out; zip -r domjudge.zip *.png;
+        run: cd ${{ github.workspace }}; mkdir out;cd organizations; for f in *; do cp "$f/logo.64x64.png" "../out/$f.64.png"; done; cd ../out; zip -r domjudge.zip *.png;
       - name: Zip logos for cds
         run: cd ${{ github.workspace }}; zip  -r out/cds.zip organizations
+      - name: Zip logos for domjudge import script
+        run: cd ${{ github.workspace }}; mkdir out/import; cd organizations; for f in *; do mkdir "../out/import/$f"; cp "$f/logo.64x64.png" "../out/import/$f/logo.64.png"; done; cd ../out/import; zip -r ../domjudge-import.zip *;
```

```suggestion
        run: cd ${{ github.workspace }}; mkdir out/import; cd organizations; for f in *; do mkdir "../out/import/$f"; cp "$f/logo.64x64.png" "../out/import/$f/logo.64x64.png"; done; cd ../out/import; zip -r ../domjudge-import.zip *;
```

Or just copy the whole folder? Which basically makes it the same as the CDS one. Maybe would be nice if we also add the JSON, so the contents of the ZIP is:

```
organizations.json
organizations/123/logo.64x64.png
...
organizations/987/logo.64x64.png
```

---

but in that case, the CDS zip can be used for the import script. I just need to add the `organizations.json` and the third zip can just be ommited.

---

Yeah I think that's best.